sharepoint/sharepoint-ps/sharepoint-online/Copy-SPOPersonalSitePage.md

```diff
@@ -0,0 +1,186 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/copy-spopersonalsitepage
+applicable: SharePoint Online
+title: Copy-SPOPersonalSitePage
+schema: 2.0.0
+author: xuyangzou
+ms.author: xuyangzou, spodeanu
+ms.reviewer:
+---
+
+# Copy-SPOPersonalSitePage
+
+## SYNOPSIS
+
+**What does this command do?**
+
+This cmdlet command allows you to relocate existing SharePoint pages by utilizing an existing copy operation. We will also copy any assets associated with the SharePoint pages to the new destination. We offer two methods for relocating pages:
+- Copy: This method keeps the original page intact while creating a duplicate at the new location.
+- Move: This method creates a new copy at the new location and deletes the original page from the source.
+
+**Where can I move the existing page(s) to and from?**
+
+| Source | Destination |
+| :------------------- | :---------- | 
+| SharePoint pages library in OneDrive  | SharePoint pages library in OneDrive | 
+| SharePoint pages library in OneDrive | SharePoint site | 
+| SharePoint site    | SharePoint site | 
+
+**How do I query the code status?**
+
+After the copy is finished, you'll see the status of the code. If the copy was successful, we'll provide the URL for the new page
```

Comment:
>status of the code [](http://example.com/codeflow?start=43&length=18)

not clear -- maybe

"After the cmdlet is run, you'll see the status code of the copy operation. If the copy was successful, we'll provide the URL for the new page."